11ty/definitions/master-slave.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Master Slave
+slug: master-slave
+defined: false
+speech: noun
+alt_words:
+  - Main & Secondary
+  - Primary & Secondary
+  - Primary & Replica
+  - Master & Minion
+  - Parent & Workers or Helpers
+  - Leader & Follower
+  - Agency & Operatives
+---
+
+A model of asymmetric communication or control where one device or process have total control over other components or are controlled by a component.
+
+## Issues
+
+Master/slave terminology is problematic given that slavery was institutionalized in the United States for hundreds of years. Yet, this type of offensive lanugage is still used casually in many programming languages.
+
+## Impact
+
+This outdated convention has strong associations with slavery which is exclusionary to members of our community who may continue to live with the negative impacts of slavery. It also demonstrates how pervasive racism is throughout our culture and lanuage.
+
+## Usage Tip
+
+There are many alternative terminologies to describe the relationship between two components. It is important to be aware of empathy as well as diversity in the world of technology to ensure everyone feels welcome at the table.
```

Comment:
```suggestion
Seek to change any existing "master/slave" terminology to one of the suggested terms, as there are many alternative terminologies to describe the relationship between two components, many of which are much more specific. Avoid creating new projects with this terminology.

Even if you may not be negatively impacted by this terminology, it can negatively impact other existing and prospective contributors onto your project. The terminology isn't so unique that it's irreplaceable; the lift tends to be trivial in comparison to the potential impact.
```